test_framework/runner.py

```diff
@@ -23,6 +23,16 @@
 from test_framework.tacky.suite import Optimizations
 
 
+class MyTextTestResult(unittest.TextTestResult):
```

Comment:
This works fine!

Now the only thing left is naming, I guess. This is a _very_ lazy name... Should we stick to it?

An alternative could be `ConciseTextTestResult` or using any other of these adjectives like brief, terse, reduced, succinct...